cli/spcli/sectors.go

```diff
@@ -1336,7 +1336,7 @@ func TerminateSectorCmd(getActorAddress ActorAddressGetter) *cli.Command {
 			},
 		},
 		Action: func(cctx *cli.Context) error {
-			if cctx.NArg() < 1 {
+			if cctx.NArg() != 1 {
```

Comment:
no, this one can take >1 arguments too, see the line starting with `sectorNumbers := lo.Map(cctx.Args().Slice(),`, undo this change